### PR TITLE
Drop `laminas/laminas-zendframework-bridge` and `zendframework/*` compatibility

### PR DESCRIPTION
**Pull request by PowerKiKi**
|    Q          |   A
|-------------- | ------
| Documentation | no
| Bugfix        | yes
| BC Break      | no
| New Feature   | no
| RFC           | no
| QA            | no

### Description

Increase performance by removing a compatibility layer while **not** introducing breaking changes.

This follow the process described in details in:

https://github.com/laminas/technical-steering-committee/blob/main/meetings/minutes/2021-08-02-TSC-Minutes.md#remove-laminaslaminas-zendframework-bridge-dependency-from-our-packages

### CODE REVIEW

**Review by Ocramius — CHANGES_REQUESTED** [review]
CI failing here - unrelated to patch, but a blocker for merges, as this component is mostly security-only

**Comment by PowerKiKi** [comment]
Tests should be fixed now

File: test/ClassFileLocatorTest.php [hunk 1]
```diff
@@ -191,8 +191,8 @@ public function testIgnoresMethodsNamedAfterKeywords()
         }, []);
 
         $expected = [
-            TestAsset\WithMethodsNamedAfterKeywords\WithReturnTypeDeclaration::class,
             TestAsset\WithMethodsNamedAfterKeywords\WithoutReturnTypeDeclaration::class,
+            TestAsset\WithMethodsNamedAfterKeywords\WithReturnTypeDeclaration::class,
```
Comment (Ocramius) [inline test/ClassFileLocatorTest.php:195]: It's a bit of a BC break, but 🤷  - no interface change.

**Comment by Ocramius** [comment]
Thanks @PowerKiKi!

**Pull request merged**